app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @user = params[:id] ? User.find(params[:id]) : current_user
+  end
```

Comment:
do you ever make a request to this action without the id param? I don't think it's possible if you follow REST conventions

Reply:
that's right, removed